test/integration_test.sh

```diff
@@ -244,6 +244,13 @@ python $script_dir/create_lira_config.py \
     --pipeline_tools_prefix $pipeline_tools_prefix > config.json
 
 # 7. Start Lira
+
+lira_container_name=lira
```

Comment:
maybe we should just hardcode lira everywhere instead of defining a variable, since we probably won't change it